source-s3/main.go

```diff
@@ -188,7 +192,7 @@ func (l *s3Listing) poll() error {
 		input.ContinuationToken = l.output.NextContinuationToken
 	}
 
-	if out, err := l.s3.ListObjectsV2(&input); err != nil {
+	if out, err := l.s3.ListObjectsV2WithContext(context.Background(), &input); err != nil {
```

Comment:
It feels like this was intended to use `ctx` rather than `context.Background()`?

Reply:
Totally! Thanks.